app/controllers/derby_events_controller.rb

```diff
@@ -7,14 +7,14 @@ def index
     if admin_signed_in?
       @derby_events = DerbyEvent.all
     else
-      @derby_events = DerbyEvent.where(approved: true, deleted: false).where("start_date >= ? OR end_date >= ?", Date.today, Date.today)
+      @derby_events = DerbyEvent.viewable.where("start_date >= ? OR end_date >= ?", Date.today, Date.today)
```

Comment:
I didn't know scopes yet; that's cool. Maybe name the date filter scope upcoming? There's also still a minor issue there regarding time zones (as per usual), but I think upcoming is clear enough. I basically want it to show events starting/ending on the current day or in the future.